Create reference tests for CSS filters

smfr opened the pull request:
#### 2602b2075a273dcbde837f1a407d5341a9b83eae
<pre>
Create reference tests for CSS filters
<a href="https://bugs.webkit.org/show_bug.cgi?id=245041">https://bugs.webkit.org/show_bug.cgi?id=245041</a>
&lt;rdar://99786910&gt;

Reviewed by Said Abou-Hallawa.

css3/filters/effect* tests were pixel tests, not reference tests. We can make reference
tests by constructing an -expected.html using the canonical SVG filters described in
&lt;<a href="https://www.w3.org/TR/filter-effects-1/">https://www.w3.org/TR/filter-effects-1/</a>&gt;.

For every CSS filter effect other than reference filters, this commit
adds a reference test for software and GPU-rendered filters. Code in `filter-helpers.js`
knows how to build the equivalent set of SVG filters given a list of CSS filters,
which is used in expectations.

Pixel tolerance is required in many tests to cover rendering differences.

* LayoutTests/TestExpectations:
* LayoutTests/css3/filters/css-opacity-with-drop-shadow-expected.html: Removed.
* LayoutTests/css3/filters/css-opacity-with-drop-shadow.html: Removed.
* LayoutTests/css3/filters/effect-blur-expected.html: Added.
* LayoutTests/css3/filters/effect-blur-expected.png: Removed.
* LayoutTests/css3/filters/effect-blur-expected.txt: Removed.
* LayoutTests/css3/filters/effect-blur-hw-expected.html: Added.
* LayoutTests/css3/filters/effect-blur-hw-expected.png: Removed.
* LayoutTests/css3/filters/effect-blur-hw-expected.txt: Removed.
* LayoutTests/css3/filters/effect-blur-hw.html:
* LayoutTests/css3/filters/effect-blur.html:
* LayoutTests/css3/filters/effect-brightness-expected.html: Added.
* LayoutTests/css3/filters/effect-brightness-expected.png: Removed.
* LayoutTests/css3/filters/effect-brightness-expected.txt: Removed.
* LayoutTests/css3/filters/effect-brightness-hw-expected.html: Added.
* LayoutTests/css3/filters/effect-brightness-hw-expected.png: Removed.
* LayoutTests/css3/filters/effect-brightness-hw-expected.txt: Removed.
* LayoutTests/css3/filters/effect-brightness-hw.html:
* LayoutTests/css3/filters/effect-brightness.html:
* LayoutTests/css3/filters/effect-combined-expected.html: Added.
* LayoutTests/css3/filters/effect-combined-expected.png: Removed.
* LayoutTests/css3/filters/effect-combined-expected.txt: Removed.
* LayoutTests/css3/filters/effect-combined-hw-expected.html: Added.
* LayoutTests/css3/filters/effect-combined-hw-expected.png: Removed.
* LayoutTests/css3/filters/effect-combined-hw-expected.txt: Removed.
* LayoutTests/css3/filters/effect-combined-hw.html:
* LayoutTests/css3/filters/effect-combined.html:
* LayoutTests/css3/filters/effect-contrast-expected.html: Added.
* LayoutTests/css3/filters/effect-contrast-expected.png: Removed.
* LayoutTests/css3/filters/effect-contrast-expected.txt: Removed.
* LayoutTests/css3/filters/effect-contrast-hw-expected.html: Added.
* LayoutTests/css3/filters/effect-contrast-hw-expected.png: Removed.
* LayoutTests/css3/filters/effect-contrast-hw-expected.txt: Removed.
* LayoutTests/css3/filters/effect-contrast-hw.html:
* LayoutTests/css3/filters/effect-contrast.html:
* LayoutTests/css3/filters/effect-drop-shadow-expected.html: Added.
* LayoutTests/css3/filters/effect-drop-shadow-expected.png: Removed.
* LayoutTests/css3/filters/effect-drop-shadow-expected.txt: Removed.
* LayoutTests/css3/filters/effect-drop-shadow-hw-expected.html: Added.
* LayoutTests/css3/filters/effect-drop-shadow-hw-expected.png: Removed.
* LayoutTests/css3/filters/effect-drop-shadow-hw-expected.txt: Removed.
* LayoutTests/css3/filters/effect-drop-shadow-hw.html:
* LayoutTests/css3/filters/effect-drop-shadow.html:
* LayoutTests/css3/filters/effect-grayscale-expected.html: Added.
* LayoutTests/css3/filters/effect-grayscale-expected.png: Removed.
* LayoutTests/css3/filters/effect-grayscale-expected.txt: Removed.
* LayoutTests/css3/filters/effect-grayscale-hw-expected.html: Added.
* LayoutTests/css3/filters/effect-grayscale-hw-expected.png: Removed.
* LayoutTests/css3/filters/effect-grayscale-hw-expected.txt: Removed.
* LayoutTests/css3/filters/effect-grayscale-hw.html:
* LayoutTests/css3/filters/effect-grayscale.html:
* LayoutTests/css3/filters/effect-hue-rotate-expected.html: Added.
* LayoutTests/css3/filters/effect-hue-rotate-expected.png: Removed.
* LayoutTests/css3/filters/effect-hue-rotate-expected.txt: Removed.
* LayoutTests/css3/filters/effect-hue-rotate-hw-expected.html: Added.
* LayoutTests/css3/filters/effect-hue-rotate-hw-expected.png: Removed.
* LayoutTests/css3/filters/effect-hue-rotate-hw-expected.txt: Removed.
* LayoutTests/css3/filters/effect-hue-rotate-hw.html:
* LayoutTests/css3/filters/effect-hue-rotate.html:
* LayoutTests/css3/filters/effect-invert-expected.html: Added.
* LayoutTests/css3/filters/effect-invert-expected.png: Removed.
* LayoutTests/css3/filters/effect-invert-expected.txt: Removed.
* LayoutTests/css3/filters/effect-invert-hw-expected.html: Added.
* LayoutTests/css3/filters/effect-invert-hw-expected.png: Removed.
* LayoutTests/css3/filters/effect-invert-hw-expected.txt: Removed.
* LayoutTests/css3/filters/effect-invert-hw.html:
* LayoutTests/css3/filters/effect-invert.html:
* LayoutTests/css3/filters/effect-opacity-expected.html: Added.
* LayoutTests/css3/filters/effect-opacity-expected.png: Removed.
* LayoutTests/css3/filters/effect-opacity-expected.txt: Removed.
* LayoutTests/css3/filters/effect-opacity-hw-expected.html: Added.
* LayoutTests/css3/filters/effect-opacity-hw-expected.png: Removed.
* LayoutTests/css3/filters/effect-opacity-hw-expected.txt: Removed.
* LayoutTests/css3/filters/effect-opacity-hw.html:
* LayoutTests/css3/filters/effect-opacity.html:
* LayoutTests/css3/filters/effect-saturate-expected.html: Added.
* LayoutTests/css3/filters/effect-saturate-expected.png: Removed.
* LayoutTests/css3/filters/effect-saturate-expected.txt: Removed.
* LayoutTests/css3/filters/effect-saturate-hw-expected.html: Added.
* LayoutTests/css3/filters/effect-saturate-hw-expected.png: Removed.
* LayoutTests/css3/filters/effect-saturate-hw-expected.txt: Removed.
* LayoutTests/css3/filters/effect-saturate-hw.html:
* LayoutTests/css3/filters/effect-saturate.html:
* LayoutTests/css3/filters/effect-sepia-expected.html: Added.
* LayoutTests/css3/filters/effect-sepia-expected.png: Removed.
* LayoutTests/css3/filters/effect-sepia-expected.txt: Removed.
* LayoutTests/css3/filters/effect-sepia-hw-expected.html: Added.
* LayoutTests/css3/filters/effect-sepia-hw-expected.png: Removed.
* LayoutTests/css3/filters/effect-sepia-hw-expected.txt: Removed.
* LayoutTests/css3/filters/effect-sepia-hw.html:
* LayoutTests/css3/filters/effect-sepia.html:
* LayoutTests/css3/filters/resources/filter-helpers.css: Added.
* LayoutTests/css3/filters/resources/filter-helpers.js: Added.
* LayoutTests/platform/ios/css3/filters/effect-blur-expected.txt: Removed.
* LayoutTests/platform/ios/css3/filters/effect-blur-hw-expected.txt: Removed.
* LayoutTests/platform/ios/css3/filters/effect-brightness-expected.txt: Removed.
* LayoutTests/platform/ios/css3/filters/effect-brightness-hw-expected.txt: Removed.
* LayoutTests/platform/ios/css3/filters/effect-combined-expected.txt: Removed.
* LayoutTests/platform/ios/css3/filters/effect-combined-hw-expected.txt: Removed.
* LayoutTests/platform/ios/css3/filters/effect-contrast-expected.txt: Removed.
* LayoutTests/platform/ios/css3/filters/effect-contrast-hw-expected.txt: Removed.
* LayoutTests/platform/ios/css3/filters/effect-drop-shadow-expected.txt: Removed.
* LayoutTests/platform/ios/css3/filters/effect-drop-shadow-hw-expected.txt: Removed.
* LayoutTests/platform/ios/css3/filters/effect-grayscale-expected.txt: Removed.
* LayoutTests/platform/ios/css3/filters/effect-grayscale-hw-expected.txt: Removed.
* LayoutTests/platform/ios/css3/filters/effect-hue-rotate-expected.txt: Removed.
* LayoutTests/platform/ios/css3/filters/effect-hue-rotate-hw-expected.txt: Removed.
* LayoutTests/platform/ios/css3/filters/effect-invert-expected.txt: Removed.
* LayoutTests/platform/ios/css3/filters/effect-invert-hw-expected.txt: Removed.
* LayoutTests/platform/ios/css3/filters/effect-opacity-expected.txt: Removed.
* LayoutTests/platform/ios/css3/filters/effect-opacity-hw-expected.txt: Removed.
* LayoutTests/platform/ios/css3/filters/effect-saturate-expected.txt: Removed.
* LayoutTests/platform/ios/css3/filters/effect-saturate-hw-expected.txt: Removed.
* LayoutTests/platform/ios/css3/filters/effect-sepia-expected.txt: Removed.
* LayoutTests/platform/ios/css3/filters/effect-sepia-hw-expected.txt: Removed.
* LayoutTests/platform/mac-wk2/css3/filters/effect-blur-expected.png: Removed.
* LayoutTests/platform/mac-wk2/css3/filters/effect-combined-expected.png: Removed.
* LayoutTests/platform/mac-wk2/css3/filters/effect-opacity-expected.png: Removed.

Canonical link: <a href="https://commits.webkit.org/254408@main">https://commits.webkit.org/254408@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/617205b4a32d5961cec658d55c18454eb43dd4da

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [❌ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/88856 "1 style error") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/69/builds/33421 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/43/builds/19738 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/8/builds/98058 "Built successfully") | [  ~~🛠 🧪 win~~](https://ews-build.webkit.org/#/builders/10/builds/154543 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/64/builds/31927 "Built successfully") | [✅ 🛠 mac-debug](https://ews-build.webkit.org/#/builders/71/builds/27532 "Built successfully") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/81100 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/12/builds/92679 "Built successfully") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/94485 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/68/builds/25335 "Passed tests") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/75839 "Built successfully") | [❌ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/35/builds/25288 "Found 2 new test failures: css3/filters/effect-contrast-hw.html, css3/filters/effect-opacity-hw.html (failure)") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/9/builds/80207 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/3/builds/80246 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/34/builds/68252 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/66/builds/29723 "Built successfully") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/73/builds/14264 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/67/builds/29453 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/70/builds/15266 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/74/builds/3085 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/63/builds/32891 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/60/builds/38199 "Passed tests") | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/65/builds/31579 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/62/builds/34371 "Passed tests") | | 
<!--EWS-Status-Bubble-End-->

shallawa commented on LayoutTests/css3/filters/resources/filter-helpers.js:
```diff
@@ -0,0 +1,210 @@
+    const color = matchResult[4];
+
+    return elementFromHTMLString(`<filter>
+        <feGaussianBlur in="SourceAlpha" stdDeviation="${stdDeviation}"/>
+          <feOffset dx="${xOffset}" dy="${yOffset}" result="offsetblur"/>
+          <feFlood flood-color="${color}"/>
+          <feComposite in2="offsetblur" operator="in"/>
+          <feMerge>
+            <feMergeNode/>
+            <feMergeNode in="SourceGraphic"/>
+          </feMerge>
```
Is there a reason we can't use 

`<feDropShadow dx="${xOffset}" dy="${yOffset}" stdDeviation="${stdDeviation}" flood-color="${color}"/>`

Also I think the effects starting from `<feOffset>` are indented one extra tab.

shallawa commented on LayoutTests/css3/filters/resources/filter-helpers.js:
```diff
@@ -0,0 +1,210 @@
+            }
+
+            if (filterType === 'drop-shadow') {
+                // Because the SVG equivalent of drop-shadow involves multiple components,
+                // we can't support it in a list.
+                filterElement = filterCreationFunction(param);
```
I think it is okay but we have to build the drop-shadow as a <filter> element not as a primitive element?

smfr committed:
Create reference tests for CSS filters
https://bugs.webkit.org/show_bug.cgi?id=245041
<rdar://99786910>

Reviewed by Said Abou-Hallawa.

css3/filters/effect* tests were pixel tests, not reference tests. We can make reference
tests by constructing an -expected.html using the canonical SVG filters described in
<https://www.w3.org/TR/filter-effects-1/>.

For every CSS filter effect other than reference filters, this commit
adds a reference test for software and GPU-rendered filters. Code in `filter-helpers.js`
knows how to build the equivalent set of SVG filters given a list of CSS filters,
which is used in expectations.

Pixel tolerance is required in many tests to cover rendering differences.

* LayoutTests/TestExpectations:
* LayoutTests/css3/filters/css-opacity-with-drop-shadow-expected.html: Removed.
* LayoutTests/css3/filters/css-opacity-with-drop-shadow.html: Removed.
* LayoutTests/css3/filters/effect-blur-expected.html: Added.
* LayoutTests/css3/filters/effect-blur-expected.png: Removed.
* LayoutTests/css3/filters/effect-blur-expected.txt: Removed.
* LayoutTests/css3/filters/effect-blur-hw-expected.html: Added.
* LayoutTests/css3/filters/effect-blur-hw-expected.png: Removed.
* LayoutTests/css3/filters/effect-blur-hw-expected.txt: Removed.
* LayoutTests/css3/filters/effect-blur-hw.html:
* LayoutTests/css3/filters/effect-blur.html:
* LayoutTests/css3/filters/effect-brightness-expected.html: Added.
* LayoutTests/css3/filters/effect-brightness-expected.png: Removed.
* LayoutTests/css3/filters/effect-brightness-expected.txt: Removed.
* LayoutTests/css3/filters/effect-brightness-hw-expected.html: Added.
* LayoutTests/css3/filters/effect-brightness-hw-expected.png: Removed.
* LayoutTests/css3/filters/effect-brightness-hw-expected.txt: Removed.
* LayoutTests/css3/filters/effect-brightness-hw.html:
* LayoutTests/css3/filters/effect-brightness.html:
* LayoutTests/css3/filters/effect-combined-expected.html: Added.
* LayoutTests/css3/filters/effect-combined-expected.png: Removed.
* LayoutTests/css3/filters/effect-combined-expected.txt: Removed.
* LayoutTests/css3/filters/effect-combined-hw-expected.html: Added.
* LayoutTests/css3/filters/effect-combined-hw-expected.png: Removed.
* LayoutTests/css3/filters/effect-combined-hw-expected.txt: Removed.
* LayoutTests/css3/filters/effect-combined-hw.html:
* LayoutTests/css3/filters/effect-combined.html:
* LayoutTests/css3/filters/effect-contrast-expected.html: Added.
* LayoutTests/css3/filters/effect-contrast-expected.png: Removed.
* LayoutTests/css3/filters/effect-contrast-expected.txt: Removed.
* LayoutTests/css3/filters/effect-contrast-hw-expected.html: Added.
* LayoutTests/css3/filters/effect-contrast-hw-expected.png: Removed.
* LayoutTests/css3/filters/effect-contrast-hw-expected.txt: Removed.
* LayoutTests/css3/filters/effect-contrast-hw.html:
* LayoutTests/css3/filters/effect-contrast.html:
* LayoutTests/css3/filters/effect-drop-shadow-expected.html: Added.
* LayoutTests/css3/filters/effect-drop-shadow-expected.png: Removed.
* LayoutTests/css3/filters/effect-drop-shadow-expected.txt: Removed.
* LayoutTests/css3/filters/effect-drop-shadow-hw-expected.html: Added.
* LayoutTests/css3/filters/effect-drop-shadow-hw-expected.png: Removed.
* LayoutTests/css3/filters/effect-drop-shadow-hw-expected.txt: Removed.
* LayoutTests/css3/filters/effect-drop-shadow-hw.html:
* LayoutTests/css3/filters/effect-drop-shadow.html:
* LayoutTests/css3/filters/effect-grayscale-expected.html: Added.
* LayoutTests/css3/filters/effect-grayscale-expected.png: Removed.
* LayoutTests/css3/filters/effect-grayscale-expected.txt: Removed.
* LayoutTests/css3/filters/effect-grayscale-hw-expected.html: Added.
* LayoutTests/css3/filters/effect-grayscale-hw-expected.png: Removed.
* LayoutTests/css3/filters/effect-grayscale-hw-expected.txt: Removed.
* LayoutTests/css3/filters/effect-grayscale-hw.html:
* LayoutTests/css3/filters/effect-grayscale.html:
* LayoutTests/css3/filters/effect-hue-rotate-expected.html: Added.
* LayoutTests/css3/filters/effect-hue-rotate-expected.png: Removed.
* LayoutTests/css3/filters/effect-hue-rotate-expected.txt: Removed.
* LayoutTests/css3/filters/effect-hue-rotate-hw-expected.html: Added.
* LayoutTests/css3/filters/effect-hue-rotate-hw-expected.png: Removed.
* LayoutTests/css3/filters/effect-hue-rotate-hw-expected.txt: Removed.
* LayoutTests/css3/filters/effect-hue-rotate-hw.html:
* LayoutTests/css3/filters/effect-hue-rotate.html:
* LayoutTests/css3/filters/effect-invert-expected.html: Added.
* LayoutTests/css3/filters/effect-invert-expected.png: Removed.
* LayoutTests/css3/filters/effect-invert-expected.txt: Removed.
* LayoutTests/css3/filters/effect-invert-hw-expected.html: Added.
* LayoutTests/css3/filters/effect-invert-hw-expected.png: Removed.
* LayoutTests/css3/filters/effect-invert-hw-expected.txt: Removed.
* LayoutTests/css3/filters/effect-invert-hw.html:
* LayoutTests/css3/filters/effect-invert.html:
* LayoutTests/css3/filters/effect-opacity-expected.html: Added.
* LayoutTests/css3/filters/effect-opacity-expected.png: Removed.
* LayoutTests/css3/filters/effect-opacity-expected.txt: Removed.
* LayoutTests/css3/filters/effect-opacity-hw-expected.html: Added.
* LayoutTests/css3/filters/effect-opacity-hw-expected.png: Removed.
* LayoutTests/css3/filters/effect-opacity-hw-expected.txt: Removed.
* LayoutTests/css3/filters/effect-opacity-hw.html:
* LayoutTests/css3/filters/effect-opacity.html:
* LayoutTests/css3/filters/effect-saturate-expected.html: Added.
* LayoutTests/css3/filters/effect-saturate-expected.png: Removed.
* LayoutTests/css3/filters/effect-saturate-expected.txt: Removed.
* LayoutTests/css3/filters/effect-saturate-hw-expected.html: Added.
* LayoutTests/css3/filters/effect-saturate-hw-expected.png: Removed.
* LayoutTests/css3/filters/effect-saturate-hw-expected.txt: Removed.
* LayoutTests/css3/filters/effect-saturate-hw.html:
* LayoutTests/css3/filters/effect-saturate.html:
* LayoutTests/css3/filters/effect-sepia-expected.html: Added.
* LayoutTests/css3/filters/effect-sepia-expected.png: Removed.
* LayoutTests/css3/filters/effect-sepia-expected.txt: Removed.
* LayoutTests/css3/filters/effect-sepia-hw-expected.html: Added.
* LayoutTests/css3/filters/effect-sepia-hw-expected.png: Removed.
* LayoutTests/css3/filters/effect-sepia-hw-expected.txt: Removed.
* LayoutTests/css3/filters/effect-sepia-hw.html:
* LayoutTests/css3/filters/effect-sepia.html:
* LayoutTests/css3/filters/resources/filter-helpers.css: Added.
* LayoutTests/css3/filters/resources/filter-helpers.js: Added.
* LayoutTests/platform/ios/css3/filters/effect-blur-expected.txt: Removed.
* LayoutTests/platform/ios/css3/filters/effect-blur-hw-expected.txt: Removed.
* LayoutTests/platform/ios/css3/filters/effect-brightness-expected.txt: Removed.
* LayoutTests/platform/ios/css3/filters/effect-brightness-hw-expected.txt: Removed.
* LayoutTests/platform/ios/css3/filters/effect-combined-expected.txt: Removed.
* LayoutTests/platform/ios/css3/filters/effect-combined-hw-expected.txt: Removed.
* LayoutTests/platform/ios/css3/filters/effect-contrast-expected.txt: Removed.
* LayoutTests/platform/ios/css3/filters/effect-contrast-hw-expected.txt: Removed.
* LayoutTests/platform/ios/css3/filters/effect-drop-shadow-expected.txt: Removed.
* LayoutTests/platform/ios/css3/filters/effect-drop-shadow-hw-expected.txt: Removed.
* LayoutTests/platform/ios/css3/filters/effect-grayscale-expected.txt: Removed.
* LayoutTests/platform/ios/css3/filters/effect-grayscale-hw-expected.txt: Removed.
* LayoutTests/platform/ios/css3/filters/effect-hue-rotate-expected.txt: Removed.
* LayoutTests/platform/ios/css3/filters/effect-hue-rotate-hw-expected.txt: Removed.
* LayoutTests/platform/ios/css3/filters/effect-invert-expected.txt: Removed.
* LayoutTests/platform/ios/css3/filters/effect-invert-hw-expected.txt: Removed.
* LayoutTests/platform/ios/css3/filters/effect-opacity-expected.txt: Removed.
* LayoutTests/platform/ios/css3/filters/effect-opacity-hw-expected.txt: Removed.
* LayoutTests/platform/ios/css3/filters/effect-saturate-expected.txt: Removed.
* LayoutTests/platform/ios/css3/filters/effect-saturate-hw-expected.txt: Removed.
* LayoutTests/platform/ios/css3/filters/effect-sepia-expected.txt: Removed.
* LayoutTests/platform/ios/css3/filters/effect-sepia-hw-expected.txt: Removed.
* LayoutTests/platform/mac-wk2/css3/filters/effect-blur-expected.png: Removed.
* LayoutTests/platform/mac-wk2/css3/filters/effect-combined-expected.png: Removed.
* LayoutTests/platform/mac-wk2/css3/filters/effect-opacity-expected.png: Removed.

Canonical link: https://commits.webkit.org/254408@main
